simple/test.js

```diff
@@ -2571,11 +2591,11 @@ exports.test = function (global) {
         },
         {
           name: "k-shortest-outbound",
-          params: { func: shortestOutbound }
+          params: { func: kShortestOutbound }
```

Comment:
I guess this is a bug, because the `shortestOutbound` function was already used above in line 2586 and the `kShortestOutbound` function was not used at all. Alternatively, I can just delete the `kShortestOutbound` function.